src/mgos_imu_icm20948.c

```diff
@@ -0,0 +1,482 @@
+/*
+ * Copyright 2019 Google Inc.
```

Comment:
(same as for the header file) - you wrote this code, so you get to have the copyright.
Please keep the license on L3-14 and you're good to go.